org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -882,11 +882,8 @@ GithubOidcNbConvertDeploy:
     ProviderRoleName: !Sub ${resourcePrefix}-${appName}-nbconvert-deploy
     MaxSessionDuration: 7200
     ManagedPolicyArns:
-      - "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryFullAccess"
-      - "arn:aws:iam::aws:policy/AWSLambda_FullAccess"
-      - "arn:aws:iam::aws:policy/CloudWatchLogsFullAccess"
-      - "arn:aws:iam::aws:policy/IAMFullAccess"
-      - "arn:aws:iam::aws:policy/AWSCloudFormationFullAccess"
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
This resource is now basically the same as `GithubOidcSageBionetworksSynapseDockerRegistry`.  Would it make sense to consolidate these to a single generic `GithubOidcSageBionetworksSynapse` resource that gives access to multiple repos?  The benefit is that when new repos pop up to deploy lambda or CDK they can just use the generic resource.

Reply:
Good idea! Renamed, added registry repo.